internal/compiler/program.go

```diff
@@ -311,10 +311,19 @@ func (p *Program) getSyntacticDiagnosticsForFile(sourceFile *ast.SourceFile) []*
 }
 
 func (p *Program) getBindDiagnosticsForFile(sourceFile *ast.SourceFile) []*ast.Diagnostic {
+	// TODO: restore this; tsgo's main depends on this function binding all files for timing.
```

Comment:
When and why would we restore this? Seems odd for `skipLibChecks` to affect binding.

Reply:
We have one function that gets both bind and check diagnostics, and both are silenced by this same function in Strada. Why? I have no idea, but that's how it was. It's just that they are split apart in Corsa for timing reasons at the moment.